packages/medusa/src/repositories/gift-card.ts

```diff
@@ -1,5 +1,48 @@
-import { EntityRepository, Repository } from "typeorm"
+import { flatten, groupBy, map, merge } from "lodash"
+import { EntityRepository, FindManyOptions, Repository } from "typeorm"
 import { GiftCard } from "../models/gift-card"
 
 @EntityRepository(GiftCard)
-export class GiftCardRepository extends Repository<GiftCard> {}
+export class GiftCardRepository extends Repository<GiftCard> {
```

Comment:
Copy-paste from order repo

Reply:
This is cool, but let's add `findOneWithRelations` too, and fix tests accordingly :)